webui.nim

```diff
@@ -258,7 +256,7 @@ proc script*(window: Window; script: string; timeout: int = 0, bufferLen: static
   let 
     error = bindings.script(csize_t window, cstring script, csize_t timeout, cast[cstring](addr buffer), csize_t bufferLen)
```

Comment:
I think it would be better to use `addr buffer[0]` for unified coding style here?

(Either may not be a safe option, through)

Reply:
@neroist is the best person to confirm that.